### ISSUE TITLE
Do you succeed in reproducing the author's result on smallNORB?

### ISSUE THREAD

**Issue by yhyu13** [shzygmyx/Matrix-Capsules-pytorch#1]
Hi, @shzygmyx 

I am a contributor to another [matrix CapsNet](https://github.com/www0wwwjs1/Matrix-Capsules-EM-Tensorflow) in TensorFlow. Are you able to reproduce the result suggested by the author of the paper?

**Comment by YuxianMeng** [comment]
I haven't try training CapsNet on SmallNORB since I'm not familiar with this dataset and I'm busy with my application for graduate school. I'd appreciate it if you can help test my program on smallNORB :)

**Comment by ZHANG-SHI-CHANG** [comment]
when i try training CapsNet on SmallNORB, it is only 20% accuracy with 1 iter, i don't know where is wrong , i will try more iter,  :)

**Comment by YuxianMeng** [comment]
@ZHANG-SHI-CHANG Thanks for your contribution :) I'm not quite sure about this, but maybe you can try to use smaller learning rate and change the lambda/m schedule

**Comment by OlivierX** [comment]
Same issue here...
